frontend/src/components/MessagingDisplay.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../styles/messaging.scss';
+
+function MessagingDisplay(props){
+
+    const containerSize = props.largeDisplay != "true" ? "message-display-container-small" :
+    "message-display-container-large"
```

Comment:
Also, the if ames are a little bit deceptive, because they tell me they only set size, while they set colors, font sizes, etc, with a lot of shared stuff. What I'd probably to is to have a generic message class, `message-container` to have the generic styles, and `error-message-container` and `plain-message-container` would set the specific CSS properties.

I mention classes and not ids because I think in theory we could have more than one message on the page in which case id is problematic (semantically for sure, but maybe actually breaking the page). No need to test it extensively, but I think there is no harm in using `className`, thus futureproofing this. :)